### ISSUE TITLE
Return DeleteResource back to codebase

### ISSUE THREAD

**Issue by cmh2166** [sul-dlss-labs/taco#306]
post #295

**Comment by jcoyne** [comment]
This feels like a duplicate of #261. I'm not sure we need a new ticket.

**Comment by cmh2166** [comment]
@aaron-collier cool? This is coming from a point in your PR being reviewed now

**Comment by aaron-collier** [comment]
I'm OK with closing this as the point of "LATER:" was that there was an outstanding PR that shouldn't block my PR but will need to be rebased once #295 is merged, so I agree that we don't actually need this ticket.

**Comment by cmh2166** [comment]
you didn't write "LATER" on that PR, so needed to confirm what work was where.
